accelerator/core/apis.h

```diff
@@ -162,7 +162,7 @@ status_t api_recv_mam_message(const iota_config_t* const iconf, const iota_clien
  * - SC_OK on success
  * - non-zero on error
  */
-status_t api_mam_send_message(const iota_config_t* const iconf, const iota_client_service_t* const service,
+status_t api_send_mam_message(const iota_config_t* const iconf, const iota_client_service_t* const service,
```

Comment:
I am confused about name scheme. Can you unify and clarify with some high-level descriptions?